.github/workflows/storage_visualization.yaml

```diff
@@ -29,6 +29,15 @@ jobs:
           credentials_json: "${{ secrets.GCP_STORAGE_VISUALIZATION_CREDENTIALS }}"
           project_id: cpg-common
 
+      - name: "gcloud docker auth"
```

Comment:
I've commented below, but I think it would be better to only use the kaleido image on the job where it's needed, then we don't need to push a docker image each time it runs.

Reply:
Are you suggesting to build the Kaleido image manually only? Then it'll go out of date with the driver image. This runs only once a week and the additional layer shouldn't cost much -- we also override the previous tag, so this won't accumulate versions.

Reply:
If it's only kaleido, why not freeze it at that version? I'll accept it either way (because IMO it doesn't matter too much either way), just mostly wanted to prompt if it was necessary.

Reply:
Oh, simply because we're not just using kaleido in isolation, but these jobs also write to blob store, upload to Slack, etc. I agree that it likely doesn't make much of a difference in practice, but just seemed more robust to always update, in case our utility libraries change.